waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -459,24 +459,58 @@ proc query*(w: WakuStore, query: HistoryQuery, handler: QueryHandlerFunc) {.asyn
   waku_store_messages.set(response.value.response.messages.len.int64, labelValues = ["retrieved"])
   handler(response.value.response)
 
+proc queryFrom*(w: WakuStore, query: HistoryQuery, handler: QueryHandlerFunc, peer: PeerInfo): Future[bool] {.async.} =
+  # sends the query to the given peer
+  # returns true if no error occurs, otherwise false
+  let connOpt = await w.peerManager.dialPeer(peer, WakuStoreCodec)
+  echo "here"
```

Comment:
Use log and more descriptive statement here, or remove

Reply:
Sorry, that was not supposed to be here, forgot to remove it.